FIX : 프런트에 쿠키 전달을 위해 배포 버전에 cors 옵션 수정#91

jn33-dev opened the pull request:
---
name: Pull Request
about: 오리진이 다를때, 쿠키 전달을 위한 CORS 설정 수정
title: FIX : 프런트에 쿠키 전달을 위해 배포 버전에 cors 옵션 수정#91
labels: 'backend', 'release', 'bug'
assignees: 최예나

---

**작업한 내용**
 #91 
이슈 참고


**REVIEW POINT**
- 프런트에서 요청 할때 설정 추가하셔야 합니다! 
- { withCredentials: true }

[cors 오리진 다를 때 쿠키 받기](https://velog.io/@code-bebop/CORS%EC%9D%98-Cookie)

close #91 

@heayounchoi, @pds0309

dy-p-code committed:
FEAT : header로 토큰을 받아서 회원정보조회 수정

dy-p-code committed:
FEAT : 카카오에서 정보 받아올 때 userId 추가하도록 수정

dy-p-code committed:
FEAT : 카카오 serializer 수정

dy-p-code committed:
FEAT : 토큰 들어오는건 테스트 완료, 레포지토리에서 정보 찾아오는 부분 수정 필요

dy-p-code committed:
FEAT : 회원정보조회 API 종료!

jn33-dev committed:
FIX : socketAuthentication 에러 메세지 이벤트명 수정 #83

Changes to be committed:
	modified:   src/games/games.gateway.ts

jn33-dev committed:
FIX : 로그인은 요청한 유저의 id가 socketIdMap에 있을 경우, db에 저장된 socket 정보를 삭제하고, 새로 로그인한 socketId로 갱신 처리 #83

Changes to be committed:
	modified:   src/games/games.gateway.ts
	modified:   src/games/players.service.ts

RESOLVE : issue #83

jn33-dev committed:
FIX : create-room / enter-room 이벤트에서, 방 비밀번호 관련 메서지 및 처리 로직 수정 #84

Changes to be committed:
	modified:   src/games/dto/create-room.request.dto.ts
	modified:   src/games/room.service.ts

RESOLVE : issue #84

jn33-dev committed:
FIX : secrete -> secret 스펠링 수정 #85

Changes to be committed:
	modified:   src/games/dto/create-room.request.dto.ts
	modified:   src/games/entities/room.entity.ts
	modified:   src/games/room.service.ts

RESOLVE : #85

dy-p-code committed:
FEAT : 토큰 관련 에러 시 문구 수정

jn33-dev committed:
Merge pull request #86 from gachimind/feature/83

FIX : socketAuthentication 에러 메세지 및 이중 로그인 처리 로직 수정 #83

jn33-dev committed:
Merge pull request #87 from gachimind/feature/84

FIX : create-room / enter-room 이벤트에서, 방 비밀번호 관련 메서지 및 처리 로직 수정 #84

jn33-dev committed:
Merge pull request #89 from gachimind/feature/85

FIX : secrete -> secret 스펠링 수정 #85

jn33-dev committed:
Merge branch 'release' of https://github.com/gachimind/backend into release

jn33-dev committed:
FIX : 프런트에 쿠키 전달을 위해 배포 버전에 cors 옵션 수정 #91

Changes to be committed:
	modified:   src/main.ts

RESOLVe : #91

dy-p-code committed:
FEAT : 유저정보 조회 시 에러 핸들링 내용 추가

dy-p-code reviewed (approved):
cors 수정해도 socket 문제 없길...제발...

jn33-dev committed:
BUILD : cors 설정 변경 후 로컬 빌드 #91

Changes to be committed:
	modified:   src/games/util/update-room.info.constructor.ts

jn33-dev committed:
FIX : cors 설정 테스트 #91

Changes to be committed:
	modified:   src/main.ts
	modified:   src/users/users.controller.ts

jn33-dev committed:
Merge branch 'feature/91' of https://github.com/gachimind/backend into feature/91

jn33-dev committed:
BUILD : 테스트 배포를 위한 로컬 빌드 #91
